sql/src/main/java/io/crate/expression/reference/sys/node/NodeVersionStatsExpression.java

```diff
@@ -47,5 +51,11 @@ public Boolean innerValue(NodeStatsContext nodeStatsContext) {
                 return nodeStatsContext.version().isSnapshot();
             }
         });
+        
+        childImplementations.put(MINIMUM_INDEX_COMPATIBILITY_VERSION,
+                                 constant(Version.CURRENT.minimumIndexCompatibilityVersion().toString()));
```

Comment:
```suggestion
                                 constant(Version.CURRENT.minimumIndexCompatibilityVersion().externalNumber()));
```

Would make it explicit which display form we're using.